### PR TITLE
Removing some fixme and adding stress test

### PR DESCRIPTION
**Pull request by yonidavidson**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -19,4 +19,7 @@ build-docker:
 test:
 	go test -v ./...
 
+stress:
+	go test  -run  Example -count 10000
```
Comment (tebeka) [inline Makefile]: Maybe convert to a benchmark?

File: example_test.go [hunk 2]
```diff
@@ -3,11 +3,12 @@ package carrow_test
 
 import (
 	"fmt"
+	"testing"
 
 	"github.com/353solutions/carrow"
 )
 
-func Example() {
+func TestExample(t *testing.T) {
```
Comment (tebeka) [inline example_test.go]: Why the rename? This is a [testable exampe](https://blog.golang.org/examples) (it checks the output matches)

Comment (yonidavidson) [reply example_test.go]: Cool didn't know this feature.

File: Makefile (hunk 1 above)
Comment (yonidavidson) [reply Makefile]: What I care about is trying to make the code "throw" some unwanted exceptions.

Comment (tebeka) [reply Makefile]: Yeah, a  benchmark will do that :)

**Pull request merged**